NuRadioReco/utilities/dataservers.py

```diff
@@ -0,0 +1,115 @@
+import requests
+import os
+import filelock
```

Comment:
filelock claims it requires python >= 3.8, is that a problem?

Reply:
Well the tests download without problems. We require 3.6 for NuRadioMC currently. Do you know alternatives? Or would you fix filelock to v3.4.1, when they dropped python 3.6 support?

Reply:
I guess pip should do the right thing and automatically download an older version if needed (and it looks like the old versions still exist on pypi), so I think this should be fine.